ghost/wallet.go

```diff
@@ -0,0 +1,20 @@
+package ghost
+
+import (
+	"gitlab.com/NebulousLabs/Sia/crypto"
+	"gitlab.com/NebulousLabs/Sia/modules"
+	"gitlab.com/NebulousLabs/Sia/types"
+)
+
+type StubWallet struct{}
+
+func (StubWallet) NewWalletAddress() (uh types.UnlockHash, err error)                       { return }
+func (StubWallet) SignTransaction(*types.Transaction, []crypto.Hash) (err error)            { return }
+func (StubWallet) UnspentOutputs(bool) (us []modules.UnspentOutput, err error)              { return }
+func (StubWallet) UnconfirmedParents(types.Transaction) (ps []types.Transaction, err error) { return }
+func (StubWallet) UnlockConditions(types.UnlockHash) (uc types.UnlockConditions, err error) { return }
+
+type StubTpool struct{}
+
+func (StubTpool) AcceptTransactionSet([]types.Transaction) (err error) { return }
+func (StubTpool) FeeEstimate() (min, max types.Currency, err error)    { return }
```

Comment:
These should not be exported. They can be cut-and-pasted into whatever test file needs them.

Reply:
We can copy-and-pate it but I think most tests need this and there would be a lot of duplications.

Reply:
Currently I see two occurrences of these stubs in `us`, and one in `muse`, which isn't enough to convince me that they need to be publicly exported. If you need them in multiple packages for your own tests, you can define exported versions in one of your repos.